vue/src/vueTextMask.js

```diff
@@ -0,0 +1,51 @@
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
+
+/*
+ * Vue Directive - Text Masking with Vanilla Text Mask
+ * https://github.com/text-mask/text-mask/tree/master/vanilla
+ *
+ * This directive binds input masking via Text Mask. Once the element is
+ * destroyed it will be unbinded. Pass an options object, same as you would
+ * normally for this plugin, as a parameter on the element being bound.
+ */
+
+let textMaskInputElement
```

Comment:
I will add it as a property of the directive itself, I've just always stayed away from adding props to directives in case of overwriting, but this name is specific enough it shouldn't matter. 

More here: https://vuejs.org/guide/custom-directive.html#Directive-Instance-Properties